packages/js-api/src/core/frigade.ts

```diff
@@ -228,6 +228,10 @@ export class Frigade extends Fetchable {
   public async getFlow(flowId: string) {
     await this.initIfNeeded()
 
+    return this.getFlowSync(flowId)
+  }
+
+  public getFlowSync(flowId: string) {
```

Comment:
Should we add `@ignore` so this method doesn't get pulled into our docs? Or at least add some docs